src/components/SharePopover/index.ts

```diff
@@ -1,4 +1,4 @@
-export {ShareOptions} from './constants';
+export * from './constants';
```

Comment:
If exporting all then members should be named with `Share*`, LayoutDirection is too common name